fiftyone/plugins/context.py

```
@@ -23,6 +25,89 @@
 logger = logging.getLogger(__name__)
 
 INIT_FILENAME = "__init__.py"
+PLUGIN_MODULE_PREFIX = "fiftyone.plugins.orgs"
+
+
+def _to_python_safe_name(name):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_to_python_safe_name`

(ANN202)